src/m3/functions.ts

```diff
@@ -26,7 +28,17 @@ import {flatmapNonCyclingFollowing} from "../utils/recursion.ts"
  * @return The type of the given {@link Feature}
  */
 export const type = (feature: Feature): FeaturesContainer | Datatype | typeof unresolved =>
-    (feature as Link | Property).type
+    (feature as (Link | Property)).type
+
+
+export const isRealProperty = (feature: Feature): feature is Property =>
```

Comment:
In this case real stands for "non derived", correct?
If this is the case I would say that the name is not obvious. Perhaps isNotDerivedProperty could be more obvious

Reply:
Correct. I have a branch/PR lined up that documents these things, and then indeed replaces `Real` with `NonDerived`.